.github/workflows/basic-ci-amd64.yaml

```diff
@@ -333,7 +333,8 @@ jobs:
         run: bash tests/integration/nerdctl/gha-run.sh run
 
       - name: Collect artifacts ${{ matrix.vmm }}
-        run: bash tests/integration/nerdctl/gha-run.sh collect-artifacts
+        if: always()
+        run: bash tests/integration/nerdctl/gha-run.sh collect-artifacts || true
```

Comment:
Hi @GabyCT, do you mind explaining better (at the commit level) why this is necessary?

Probably I'm missing things, but won't  this `|| true` make it difficult to debug the job? since it will mask the failed jobs?

Reply:
@beraldoleal changes applied

Reply:
Hi @GabyCT sorry if is too noise, but IIUC, this might generate errors to the output when the script fails for some reason, but it will tell GH actions the step was successfully. As a developer debugging issues in a run, I would except to see the steps failing. Any reason for not using `continue-on-error: true`?

Also, "This will ensure that we collect and archive system's data for the purpose of debugging." not sure if this is true since it is not fixing the "collect and archive"  aspect, but just making sure the GH will not fail.

Reply:
@beraldoleal changes applied